pkg/cmd/mom/output_resources_command.go

```diff
@@ -38,6 +39,8 @@ func runOutputResources(ctx context.Context) (*libraryoutputresources.OutputReso
 				libraryoutputresources.ExactRoleBinding("openshift-config-managed", "system:openshift:oauth-servercert-trust"),
 
 				libraryoutputresources.ExactPDB("openshift-oauth-apiserver", "oauth-apiserver-pdb"),
+
+				libraryoutputresources.ExactResource(apiextensionsv1.SchemeGroupVersion.Group, apiextensionsv1.SchemeGroupVersion.Version, "customresourcedefinitions", "", "rolebindingrestrictions.authorization.openshift.io"),
```

Comment:
UserWorkloadResources

Reply:
Should be updated in https://github.com/openshift/cluster-authentication-operator/pull/748/commits/bca7a996c65d0cd7ae33691bdc1c98b7fb892661